components/public-api/gitpod/experimental/v1/workspaces.proto

```diff
@@ -145,17 +145,26 @@ message WorkspaceStatus {
 message WorkspaceContext {
     // GitProvider describes the git provider
     message GitProvider {
-        // type is the git provider type, e.g. 'github', 'gitlab', 'bitbucked'
+        // type is the git provider type, e.g. 'github', 'gitlab', 'bitbucket'
         string type = 1;
 
         // hostname is the git provider hostname
         string hostname = 2;
     }
 
+    // Repository describes the originating repository
+    message Repository {
```

Comment:
Includes everything but `host` from server API. That is because we include the hostname through `GitProvider`

Reply:
I wonder whether we overexpose? Do you know which properties you need for which use cases? Maybe don't expose prematurely.

Reply:
Maybe? It's hard to say, to be honest, but we can trim it down to just `name` and `owner` if we want to limit to what we actually need.

Reply:
Agree let's just expose the minimum required

Reply:
@filiptronicek let's rather do quick small PRs adding properties when needed, if you think only name and owner then remove and unhold. PR is approved.